### PR TITLE
[IMP] point_of_sale,*: add urbanpiper setting

### PR DESCRIPTION
**Pull request by meji-odoo**
*=pos_restaurant

After this commit:
=====
- Added fields module_pos_urban_piper and pos_module_pos_urban_piper for configuration.
- Added sound when new order is received.
- Added general setting block for food delivery.

task- 4040542

Co-authored-by: paso-odoo <paso@odoo.com>
Co-authored-by: shag-odoo <shag@odoo.com>

Related: https://github.com/odoo/enterprise/pull/66733
Related: https://github.com/odoo/upgrade/pull/6327

### CODE REVIEW

**Review by caburj — CHANGES_REQUESTED** [review]

**Comment by meji-odoo** [comment]
@odoo/rd-security 
Need to override security.

Reference: https://github.com/odoo/enterprise/wiki/security#JS004

> Markup and markup serve to whitelist strings as "raw markup" for t-out. Ultimate risks are similar to JS002, but generally easier to audit. Markup / markup whitelisting should be as close to the source as possible to make this auditing possible.

The product.template field self_order_description has been moved from the pos_self_order module to the point_of_sale module and renamed to public_description.

**Review by caburj — APPROVED** [review]

File: addons/point_of_sale/models/pos_config.py [hunk 1]
```diff
@@ -158,6 +158,7 @@ def _get_customer_display_types(self):
     limit_categories = fields.Boolean("Restrict Categories")
     module_pos_restaurant = fields.Boolean("Is a Bar/Restaurant")
     module_pos_discount = fields.Boolean("Global Discounts")
+    module_pos_urban_piper = fields.Boolean(string='Is an Urbanpiper')
```
Comment (mart-e) [inline addons/point_of_sale/models/pos_config.py:161]: why is this in community? the module is enterprise

**Review by mart-e — COMMENTED** [review]
> The product.template field self_order_description has been moved from the pos_self_order module to the point_of_sale module and renamed to public_description.

@meji-odoo that doesn’t answer the question why it’s safe (the answer is : because the field is sanitized)

@robodoo override=ci/security

**Comment by caburj** [comment]
robodoo r+

**Pull request closed without merging**